converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/Worker/DicomCastMeter.cs

```diff
@@ -0,0 +1,38 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Diagnostics.Metrics;
+using EnsureThat;
+
+namespace Microsoft.Health.DicomCast.Core.Features.Worker;
+public sealed class DicomCastMeter : IDisposable
+{
+    private readonly Meter _meter;
+
+    [SuppressMessage("Reliability", "CA2000:Dispose objects before losing scope", Justification = "Disposal is done through Dispose method.")]
+    public DicomCastMeter()
+        : this(new Meter("Microsoft.Health.DicomCast.Core.Features.Worker", "1.0"))
+    {
+    }
+
+    internal DicomCastMeter(Meter meter)
```

Comment:
Do we need this constructor which takes a Meter param? It is already a specific meter class.

Reply:
No, we don't require this. I have combined them into a single constructor.